static/app/views/performance/landing/views/backendView.tsx

```diff
@@ -55,10 +52,6 @@ export function BackendView(props: BasePerformanceViewProps) {
       ...[PerformanceWidgetSetting.MOST_REGRESSED, PerformanceWidgetSetting.MOST_IMPROVED]
     );
   }
-
-  if (showSpanOperationsWidget) {
-    doubleChartRowCharts.unshift(PerformanceWidgetSetting.SPAN_OPERATIONS);
-  }
```

Comment:
are we removing span ops widget?

Reply:
Yeah, span op breakdown is included, it links to span summary but the breakdown isn't repeated there, I suspect many people have trouble using it, but we can revert if that's not the case.